pages/library-card/new/index.tsx

```diff
@@ -46,26 +49,62 @@ function HomePage() {
         <title>{appTitle} | NYPL</title>
         <link rel="icon" type="image/png" href={favIconPath} />
         <meta name="viewport" content="width=device-width, initial-scale=1" />
-        <meta name="description" content="With a library card you get free access to resources and services across all New York Public Library locations." />
-        <meta name="keywords" content="NYPL, The New York Public Library, Manhattan, Bronx, Staten Island" />
-        <meta name="rights" content={`© ${new Date().getFullYear()} The New York Public Library`} />
-        <meta property="og:title" content="Apply for a library card from NYPL" />
-        <meta property="og:description" content="Get free access to resources and services with a New York Public Library card." />
+        <meta
+          name="description"
+          content="With a library card you get free access to resources and services across all New York Public Library locations."
```

Comment:
I realize this is not from this PR (and also that it is very nitpicky), but the syntax seems slightly weird to me; comma after "card"?  (Or even just "A library card gives you free access"...?)

Reply:
That's the existing copy that Courtney probably helped write... so it has gone through some scrutiny. The good thing is that this can be updated soon.